src/components/search-box.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+import { observer } from 'mobx-react'
+
+export default @observer class SearchBox extends Component {
+  onInputChange = event => {
+    this.props.artistsStore.searchedArtist = event.target.value
+  }
+
+  render() {
+    const { artistsStore } = this.props
+
+    return (
+      <div>
+        <div className="search-box__input-container">
+          <span className="search-box__artist-input-wrapper">
+            <input
+              className="search-search-box__artist-input"
+              type="text"
+              name="artist"
+              value={artistsStore.searchedArtist}
+              onChange={this.onInputChange}
+              // Applied with each typed letter
+              onSubmit={axios
```

Comment:
Zdefiniuj to jako metodę `onSubmit`. Tak to niepotrzebnie zaśmiecasz render.

Reply:
done